src/parser/jobs/rdm/modules/ManaStackGauge.tsx

```diff
@@ -0,0 +1,214 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import {JOBS} from 'data/JOBS'
+import {Event, Events} from 'event'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {CounterGauge, Gauge as CoreGauge} from 'parser/core/modules/Gauge'
+import {Statistics} from 'parser/core/modules/Statistics'
+import Suggestions, {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {Gauge} from 'parser/jobs/rdm/modules/Gauge'
```

Comment:
I would prefer to see the Black/White Gauge module renamed appropriately to help further disambiguate it from the Mana Stack Gauge.

Reply:
I'll rename it ManaGuage, and all dependency declarations of gauge to manaGauge!